SingularityUI/app/collections/TaskSearchResults.coffee

```diff
@@ -0,0 +1,63 @@
+Collection = require './collection'
+
+TaskSearchResult = require '../models/TaskSearchResult'
+
+class TaskSearchResults extends Collection
+
+    model: TaskSearchResult
+
+    params: ->
+    	anyParams = false
+    	base = '?'
+    	if @deployId
+    		base += 'deployId=' + @deployId
```

Comment:
Let's make sure these are all spaces not tabs, it's burned us earlier when testing this out. Comment applies for all files